modules/mono/editor/GodotTools/GodotTools.ProjectEditor/default.csproj.template

```diff
@@ -0,0 +1,67 @@
+<Project Sdk="Godot.NET.Sdk/%GODOT_NET_SDK%">
+  <PropertyGroup>
+    <RootNamespace>%ROOT_NAMESPACE%</RootNamespace>
+  </PropertyGroup>
+  <PropertyGroup Condition="$(RuntimeIdentifier.Contains('ios'))">
+    <TargetFramework>net8.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="!$(RuntimeIdentifier.Contains('ios'))">
+    <TargetFramework>net6.0</TargetFramework>
+    <EnableDynamicLoading>true</EnableDynamicLoading>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$(RuntimeIdentifier.Contains('ios'))">
+    <PublishAot>true</PublishAot>
+    <PublishAotUsingRuntimePack>true</PublishAotUsingRuntimePack>
+    <RunAOTCompilation>true</RunAOTCompilation>
+    <ForceFullAOT>true</ForceFullAOT>
+    <SelfContained>true</SelfContained>
```

Comment:
These should not be needed: RunAOTCompilation and ForceFullAOT only means something on Mono. SelfContained is implied to be true for PublishAot.